tools/integration_tests/interrupt/interrupt_test.go

```diff
@@ -57,14 +56,16 @@ func TestMain(m *testing.M) {
 	setup.SetUpTestDirForTestBucketFlag()
 
 	// Set up flags to run tests on.
-	mountConfig := config.MountConfig{
-		LogConfig: config.LogConfig{
-			Severity:        config.TRACE,
-			LogRotateConfig: config.DefaultLogRotateConfig(),
+	yamlContent1 := map[string]interface{}{
+		"file-system": map[string]interface{}{
+			"ignore-interrupts": true,
 		},
 	}
-	flags := [][]string{{"--implicit-dirs=true"},
-		{"--config-file=" + setup.YAMLConfigFile(mountConfig, "config.yaml")}}
+	yamlContent2 := map[string]interface{}{} // test default
+	flags := [][]string{
+		{"--implicit-dirs=true"},
```

Comment:
How do we make sure we are logging at `TRACE` level?

Reply:
In all the tests, we pass debug-fuse flag regardless which ensures that logs are written at TRACE level.
Code ref:
- https://github.com/GoogleCloudPlatform/gcsfuse/blob/165194ee04a8dfa4d08b57bf4e6f2d541482be16/tools/integration_tests/util/mounting/dynamic_mounting/dynamic_mounting.go#L41
- https://github.com/GoogleCloudPlatform/gcsfuse/blob/165194ee04a8dfa4d08b57bf4e6f2d541482be16/tools/integration_tests/util/mounting/only_dir_mounting/only_dir_mounting.go#L34
- https://github.com/GoogleCloudPlatform/gcsfuse/blob/165194ee04a8dfa4d08b57bf4e6f2d541482be16/tools/integration_tests/util/mounting/persistent_mounting/perisistent_mounting.go#L50
- https://github.com/GoogleCloudPlatform/gcsfuse/blob/165194ee04a8dfa4d08b57bf4e6f2d541482be16/tools/integration_tests/util/mounting/static_mounting/static_mounting.go#L35

Reply:
Ohh, it was redundant before.